### PR TITLE
Basic player score styles

### PR DESCRIPTION
**Pull request by beccarothwell**
Screenshot 2024-01-07 010550
Screenshot 2024-01-07 010615

### fonts folder

PressStart2P-Regular.ttf
- font file downloaded from Google fonts and added to new fonts folder

PressStart2P-Regular.scss 
- font declaration added for import elsewhere

### src folder

index.scss
- PressStart2P-Regular.scss imported at the top of the file
- flex-direction: column added to body styles

main.tsx
- index.scss import moved above component imports to ensure index.scss styles are enqueued first and do not overwrite component styles

App.tsx
- PlayerScore components wrapped in player-scores div to allow for positioning on either side of the page

App.tsx
- Styles added for player-scores wrapper div and #root set to width: 100%

### styles folder

_functions.scss
- shade and tint functions added for shading and tinting colours

_variables.scss
- additional colour variables added

#### RoundTrackerPip

RoundTrackerPip.scss
- styles added to show pips, yellow for a win, blue for a loss, grey for unplayed, and a pokeball for current round, with light and dark mode variations.

RoundTrackerPip.tsx
- class names refactored

RoundTrackerPip.tests.tsx
- tests updated with refactored class names

#### RoundTracker

RoundTracker.scss
- styles added to remove list styles and display flex

RoundTracker.tsx
- class names refactored

RoundTracker.tests.tsx
- tests updated with refactored class names

#### TextInput

TextInput.scss
- base styles added

TextInput.tsx
- label wrapped in div to allow className passed through props to wrap the whole element and apply override styles

TextInput.tests.tsx
- tests updated to reflect altered structure of return elements

#### PlayerScore

PlayerScore.scss
- layout styles added using display: grid

PlayerScores.tsx
- structure of returned elements updated to fit with use of display grid

### CODE REVIEW

File: client/src/App.tsx [hunk 1]
```diff
@@ -1,37 +1,77 @@
...
     initialiseGame(DEFAULT_PLAYERS, DEFAULT_ROUNDS)
   );
   const player1 = game.players[0];
   const player2 = game.players[1];
 
+  const [inputPlayerNames, setInputPlayerNames] = useState<PlayerNameAndId[]>(
+    getInitialPlayerNames(game.players)
+  );
+
+  const updatePlayerName = (id: string, value: string) => {
```
Comment (jennysivapalan) [inline client/src/App.tsx:22]: I wasn't expecting this code in this PR. Maybe you want to do a compare of this branch against https://github.com/TeamTopTrumps/top-trumps-game/pull/26 ?

Comment (beccarothwell) [reply client/src/App.tsx:22]: I think it's because it branches off the update-player-score branch? Not off main? So that one needs to be merged first?

Comment (jennysivapalan) [reply client/src/App.tsx:22]: yeah you can also set your PR to be reviewed off another branch (other than main) so we can see the smaller set of changes. 
I'll look over #26 shortly anyway

Comment (beccarothwell) [reply client/src/App.tsx:22]: Ah sorry I didn't know that! Is there a way for me to change it now? Or should I close this PR and try and open a new one to be reviewed off update-player-score?

**Pull request closed without merging**